/**:
  ros__parameters:
    # -- publishing period --
    curvature_interval_length: 5.0
    prevent_zero_division_value: 0.001
    odom_interval: 0
    acceptable_max_distance_to_waypoint: 2.0
    low_pass_filter_gain: 0.95
    acceptable_max_yaw_difference_rad: 1.0472
